### PR TITLE
Solves #272 - added description parameter to bq_field function.

### PR DESCRIPTION
**Pull request by byapparov**

### CODE REVIEW

File: tests/testthat/test-bq-table.R [hunk 1]
```diff
@@ -134,3 +134,19 @@ test_that("can round trip data frame with list-cols", {
   df2 <- as.data.frame(df2)
   expect_equal(df1, df2)
 })
+
+test_that("can create table with time partitioning", {
```
Comment (hadley) [inline tests/testthat/test-bq-table.R]: Shouldn't this be a test for setting the description?

Comment (byapparov) [reply tests/testthat/test-bq-table.R]: I have updated the description of the test, not sure if you would prefer to have two separate tests for this.

Comment (hadley) [reply tests/testthat/test-bq-table.R]: I don't quite understand why time partitioning is in here at all? How is it related to the rest of this PR?

Comment (byapparov) [reply tests/testthat/test-bq-table.R]: You are right, this PR is not the best place for it. Hence removed the partitioning part from the test. 

At the same time I am very keen to have some regression testing for the ability to crate tables with partitions and to have this ability documented. Any suggestions?

**Pull request merged**